browser/model/jbds.js

```diff
@@ -7,6 +7,8 @@ let ipcRenderer = require('electron').ipcRenderer;
 let Glob = require("glob").Glob;
 let replace = require("replace-in-file");
 
+var pjson = JSON.parse(fs.readFileSync('./package.json'));
```

Comment:
I'll create Util.loadJsonFile(path) where path is relative path to json file from project root. It will check two possible locations for resources: development and runtime execution. Similar what is implemented for requirements.json loading here https://github.com/redhat-developer-tooling/developer-platform-install/blob/master/browser/main.js#L79.